src/app/core/pusher/pusher.service.ts

```diff
@@ -321,7 +321,7 @@ export class PusherService implements OnDestroy {
       if (this.isCurrentWorkspace(data)) {
         this.store$.dispatch(
           new DocumentsAction.Get({
-            query: {stems: [{collectionId: data.object.id}], page: 0, pageSize: 100},
+            query: {stems: [{collectionId: data.object.id}], page: 0, pageSize: 200},
```

Comment:
200 is new magic number?

Reply:
100 was too little. It did not read the whole collection. I would even make it 500. The problem is that paging does not work properly. Remaining data are never read. WDYT?

Reply:
I think we should implement paging or read the whole collection.